packages/devtools/web/index.html

```diff
@@ -35,6 +35,7 @@
     <link rel="stylesheet" href="packages/devtools/src/logging/logging.css">
     <link rel="stylesheet" href="packages/devtools/src/performance/performance.css">
     <link rel="stylesheet" href="packages/devtools/src/profiler/profiler.css">
+    <link href="packages/devtools/src/settings/settings.css" rel="stylesheet">
     <link rel="stylesheet" href="packages/devtools/src/timeline/timeline.css">
 
     <script type="text/javascript">
```

Comment:
if you scroll down to line 97 of this file you will see where there is some pre-rendered text for the links to the app screens. Go ahead and add your entry for the settings screen to that list.

Reply:
As you mentioned that the icon looks a bit confusing where it is currently, should I add this before the • or after?

Before:
![image](https://user-images.githubusercontent.com/25892469/63044047-9ce58d80-bef7-11e9-9052-499d6d982ea3.png)

After:
![image](https://user-images.githubusercontent.com/25892469/63043981-7fb0bf00-bef7-11e9-8db0-4c042f223276.png)


I'll leave it after the • for now so that it doesn't look like part of the Logging menu

Reply:
after looks great. With it after the  • it is clear the settings icon is not part of Logging.

Reply:
Apparently there's a problem if I tried adding the gear icon in the same div with the send feedback button. It would be always there when we are not on the homescreen

![image](https://user-images.githubusercontent.com/25892469/63045056-be477900-bef9-11e9-8809-5948226aad52.png)

I decided to make a new • in the pre-rendered text for the links to the app screens so that it looks like this:
![image](https://user-images.githubusercontent.com/25892469/63045113-de773800-bef9-11e9-827d-bcab9ebe8212.png)

What do you think? Is this fine?